hw_7/task2.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+/* Результатом работы вашей функции должен быть массив пользователей. 
+У каждого объекта пользователя должно появиться поле myPosts: [<в этом массиве будут посты, 
+    которые сделал этот пользователь>]. Поле - массив, так как у одного пользователя может быть более 1-го поста.
+ */
+
+const POSTS_REQUEST = "https://jsonplaceholder.typicode.com/posts";
+const USERS_REQUEST = "https://jsonplaceholder.typicode.com/users";
+
+const getData = async () => {
+  try {
+    const postsListRequest = await fetch(POSTS_REQUEST).then((response) =>
+      response.json()
+    );
+
+    const usersListRequest = await fetch(USERS_REQUEST).then((response) =>
```

Comment:
you are using promise.all further. But in your code you wait for posts, then wait for users, the wait for users and posts together. Don't use await keyword here and above, to avoid 4 request instead of 2